### DISCUSSION TITLE
Can I clone an element?

### DISCUSSION THREAD

**Discussion by RealPeha** [solidjs/solid#794] [Q&A] [answered]
Using React I can do this:
```js
const SuperPuperWrapper = ({ children }) => {
  return React.cloneElement(children, {
    someProp: 42
  })
}
```
Sometimes this feature helps. But I don't see how I can do this in SolidJS

**Comment by orenelbaum** [comment]
What exactly are you trying to achieve? This works differently in Solid cause there are no V-DOM elements, when you do `const par = <p>hello</p>` for example, `par` will contain an actual DOM element.

**Comment by ryansolid** [comment, marked as answer]
Relevant discussion: https://github.com/solidjs/solid/issues/252
